### PR TITLE
Add IsRoot method to comment

### PR DESCRIPTION
**Pull request by Atrox**

### CODE REVIEW

File: reddit/data.go [hunk 1]
```diff
@@ -35,6 +37,12 @@ type Comment struct {
 	Distinguished string `mapstructure:"distinguished"`
 }
 
+// IsRoot is true when the comment is a top level comment.
+func (c *Comment) IsRoot() bool {
```
Comment (turnage) [inline reddit/data.go]: Based on this comment I think IsTopLevel would be a more appropriate name; more Reddit users could intuit what that means.

**Comment by turnage** [comment]
Thank you for submitting this PR! Looks great!

File: reddit/data.go (hunk 1 above)
Comment (Atrox) [inline reddit/data.go]: Definitively. I took the name from the python based wrapper and thought they maybe had their reasons for the naming but I can of course change it.

https://github.com/praw-dev/praw/blob/master/praw/models/reddit/comment.py#L16

**Comment by turnage** [comment]
Would you mind [signing the CLA](https://cla-assistant.io/turnage/graw?pullRequest=22)?

**Pull request merged**